src/docs/030_news/2021/2021-11-23-Open-Source-Camp.adoc

```diff
@@ -22,19 +22,12 @@
 [.center]
 image::https://cyberland.ijug.eu/assets/logo/camp-logo.png[]
 
-The next https://cyberland.ijug.eu/2021-12-open-source-camp/[Cyberland Open Source Camp] is just around the corner.
-This time, https://doctoolchain.org[docToolchain] will be one of the featured open source projects for this event.
+The next https://cyberland.ijug.eu/2021-12-open-source-camp/[Cyberland Open Source Camp] is just around the corner. This time, https://doctoolchain.org/[docToolchain] will be one of the featured open source projects! Calling all contributors and supporters of the project to think about the tasks we could realistically solve in those few hours. Here are some ideas from our https://github.com/docToolchain/docToolchain/issues[list of open issues] to get your brains into gear:
```

Comment:
just a hint: Git compares changes line by line. That is why - when you use the docs-as-code approach, you should write each sentence in its own line. As a result, you will have a better Diff.
The example above is wuite good, because git now does not show me the change for the second line. I have to manually compare it.

Reply:
"to get your brains into gear" I like this writing style very much!